WireAPI/Sources/WireAPI/APIs/TeamsAPI/TeamsAPIV0.swift

```diff
@@ -27,8 +27,12 @@ class TeamsAPIV0: TeamsAPI {
         self.httpClient = httpClient
     }
 
+    var apiVersion: APIVersion {
+        .v0
+    }
+
     func basePath(for teamID: Team.ID) -> String {
-        "/teams/\(teamID.transportString())"
+        "\(pathPrefix)/teams/\(teamID.transportString())"
     }
```

Comment:
For me this looks wrong because it changes the URL for `TeamsAPIV0`:
`/teams/$TEAMID` before
`/v0/teams/$TEAMID` now

Reply:
I was wrong, it's correct now